src/components/App.js

```diff
@@ -1,16 +1,47 @@
-import React from "react";
-import SushiContainer from "./SushiContainer";
-import Table from "./Table";
+import React, { useState, useEffect } from "react"
+import SushiContainer from "./SushiContainer"
+import Table from "./Table"
 
-const API = "http://localhost:3001/sushis";
+const API = "http://localhost:3001/sushis"
 
 function App() {
+  const [sushiList, setAllSushi] = useState([])
+  const [eatenSushis, setEatenSushis] = useState([])
+  const [remainingMoney, setRemainingMoney] = useState(100)
+
+  const getSushi = () => {
+    fetch(API)
+      .then((res) => res.json())
+      .then((sushi) => {
+        setAllSushi(sushi)
+      })
+  }
+
+  useEffect(() => {
+    getSushi()
+  }, [])
+
+  function handleEatSushi(sushi) {
+    if (remainingMoney >= sushi.price && !eatenSushis.includes(sushi.id)) {
```

Comment:
If you store each side of the operator in a constant, this could become much more readable:
```suggestion
    const hasSufficientFunds = remainingMoney >= sushi.price
    const isUneaten = !eatenSushis.includes(sushi.id)
    
    if (hasSufficientFunds && isUneaten) {
```